docs/sdk-upgrade.md

```diff
@@ -87,6 +87,13 @@ Steps:
  
     Set the default to the new SDK version and the alternate to the old version. (That way, the new version is in place if the A/B testing goes well.)
     
+   **Note**: For testing in `staging`, `idv_acuant_sdk_upgrade_a_b_testing_enabled` can be set to `false` like following to test the new SDK version:
```

Comment:
Since we've had questions in standup about how long to set traffic to the new version at 100% in staging, I think it would be good to answer that question in documentation.  I'd vote for one or two weeks at 100% in staging, with a preference for one week.  Thoughts?  

I'm tagging @daviddsilvanava in case he wants to chime in here.

Reply:
@eileen-nava updated with some testing coverage description.